auacm/app/modules/submission_manager/views.py

```diff
@@ -60,11 +60,28 @@ def submit():
 
     attempt.commit_to_session()
 
-    directory = directory_for_submission(attempt)
-    os.mkdir(directory)
-    uploaded_file.save(os.path.join(directory, uploaded_file.filename))
 
-    judge.Judge(attempt, uploaded_file, time_limit).run_threaded()
+    submission_path = os.path.join(app.config['DATA_FOLDER'],
+                                   'submits', str(attempt.job))
+    os.mkdir(submission_path)
+    uploaded_file.save(os.path.join(submission_path, uploaded_file.filename))
+
+    def update_status(status, test_number):
+        """Updates the status of the submission and notifies the clients that
+        the submission has a new status.
+        """
+        attempt.update_status(DB_STATUS[status])
+        Flasknado.emit('status', {
+            'submissionId': attempt.job,
+            'problemId': attempt.pid,
+            'username': attempt.username,
+            'submitTime': attempt.submit_time,
+            'testNum': test_number,
+            'status': DB_STATUS[status]
```

Comment:
Good catch. I went ahead and changed the way that this was written to make more sense. The `EVENT_STATUS` variable is back, but it's a map from database status to event status rather than relying on some sort of array indexing.